### ISSUE TITLE
Could not resolve com.facebook.react:react-native:+

### ISSUE THREAD

**Issue by edohartono** [gusparis/react-native-month-year-picker#134]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-month-year-picker@1.8.2` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-month-year-picker/android/build.gradle b/node_modules/react-native-month-year-picker/android/build.gradle
index 2455914..c57cbf0 100644
--- a/node_modules/react-native-month-year-picker/android/build.gradle
+++ b/node_modules/react-native-month-year-picker/android/build.gradle
@@ -112,7 +112,6 @@ afterEvaluate { project ->
         source = android.sourceSets.main.java.srcDirs
         classpath += files(android.bootClasspath)
         project.getConfigurations().getByName('implementation').setCanBeResolved(true)
-        classpath += files(project.getConfigurations().getByName('implementation').asList())
 
         include '**/*.java'
     }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by onnij** [comment]
I think the actual root cause is probably because of this:

https://github.com/facebook/react-native/issues/35210

**Comment by gusparis** [comment]
Fixed on #140. Can you guys check?
